src/core.ts

```diff
@@ -589,12 +601,36 @@ export function stressSolution(path: string, times: number) {
     let solCompileResult = compileCode(sol, out);
     if (solCompileResult.status !== 0) {
         vscode.window.showErrorMessage(`Compilation Error. ${sol}`);
+        let error_path = join(path, "stderr");
```

Comment:
Since `stderr` is not an important file for the user store it on `attic` instead.

```
let error_path = join(path, ATTIC, "stderr");
```